helpers/constants.js

```diff
@@ -18,15 +18,18 @@ const MANAGER_RATING = 2;
 const WORKER_RATING = 3;
 const RATING_MULTIPLIER = { 1: -1, 2: 1, 3: 1.5 };
 const SECONDS_PER_DAY = 86400;
-const RATING_1_SALT = web3Utils.soliditySha3(getRandomString(10));
-const RATING_2_SALT = web3Utils.soliditySha3(getRandomString(10));
+const RATING_1_SALT = web3Utils.soliditySha3(shortid.generate());
+const RATING_2_SALT = web3Utils.soliditySha3(shortid.generate());
```

Comment:
When you rebase this pls use the `web3Utils` imports at the top rather than explicit references.

Reply:
Kk